goss-testing/scripts/check_k8s_loadbalancers.sh

```diff
@@ -105,4 +146,23 @@ else
             if [[ $ip =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$ ]]; then echo 'PASS'; else echo 'FAIL'; fi
         fi
     done
+else
+# CHN is not configured and CAN is configured
```

Comment:
How do we know CAN is configured?   It isn't checking can_sls_check.

It seems like you want something like this:
```
if CMN configured; then
   check CMN services
   if CHN configured; then 
       check CHN services
   fi
   if CAN configured; then
       check CAN services
   fi
fi
```